src/rubocop/RubocopRunner.js

```diff
@@ -0,0 +1,55 @@
+'use babel'
+
+import { exec, findAsync } from 'atom-linter'
+
+const CFG_FILE = '.rubocop.yml'
+
+const TIMEOUT_ERROR_MSG = 'Process execution timed out'
+const LINTER_TIMEOUT_INFO_MSG = 'Linter-Rubocop: Linter timed out'
+const LINTER_TIMEOUT_INFO_DESC = 'Make sure you are not running Rubocop with a slow-starting interpreter like JRuby. '
+                                 + 'If you are still seeing timeouts, consider running your linter `on save` and not `on change`, '
+                                 + 'or reference https://github.com/AtomLinter/linter-rubocop/issues/202 .'
+
+
+function buildBaseExecutionOpts(filePath, extraOptions = {}) {
+  const baseExecOptions = {
+    cwd: atom.project.relativizePath(filePath)[0],
```

Comment:
⚠️ This should be done separately and checked for a valid return value, [`relativizePath`](https://atom.io/docs/api/v1.38.2/Project#instance-relativizePath) can return `null` if the path wasn't found in the project.